onebusaway-android/src/main/java/org/onebusaway/android/app/Application.java

```diff
@@ -70,6 +73,9 @@
     // Region preference (long id)
     private static final String TAG = "Application";
 
+    public static final String CHANNEL_REALTIME_ID = "ch101";
+    public static final String CHANNEL_BUS_ARRIVAL_ID = "ch102";
```

Comment:
Could you please change the name of `CHANNEL_BUS_ARRIVAL_ID` to `CHANNEL_ARRIVAL_REMINDERS_ID`?